diff --git a/src/main/java/gr/cti/nts/keycloak/idp/social/gsis/GsisAbstractIdentityProvider.java b/src/main/java/gr/cti/nts/keycloak/idp/social/gsis/GsisAbstractIdentityProvider.java
index 35f278d..aa00945 100644
--- a/src/main/java/gr/cti/nts/keycloak/idp/social/gsis/GsisAbstractIdentityProvider.java
+++ b/src/main/java/gr/cti/nts/keycloak/idp/social/gsis/GsisAbstractIdentityProvider.java
@@ -195,7 +195,7 @@ public abstract class GsisAbstractIdentityProvider extends AbstractOAuth2Identit
     return userSession.getNote(FEDERATED_ID_TOKEN);
   }
 
-  protected class OIDCEndpoint extends Endpoint {
+  protected static class OIDCEndpoint extends Endpoint {
     public OIDCEndpoint(AuthenticationCallback callback, RealmModel realm, EventBuilder event,
         AbstractOAuth2IdentityProvider provider) {
       super(callback, realm, event, provider);
